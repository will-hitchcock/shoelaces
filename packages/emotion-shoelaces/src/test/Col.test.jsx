import React from 'react';
import { shallow } from 'enzyme';
import { Col } from '..';

describe('<Col />', () => {
  it('should render a <Col />', () => {
    expect(shallow(<Col />)).toMatchSnapshot();
  });
});
