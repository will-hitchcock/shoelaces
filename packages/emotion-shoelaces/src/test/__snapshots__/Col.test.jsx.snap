// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Col /> should render a <Col /> 1`] = `
<styled.div
  _css={
    Object {
      "alignContent": "flex-start",
      "alignItems": "stretch",
      "alignSelf": "auto",
      "boxSizing": "border-box",
      "display": "flex",
      "flex": "0 0 auto",
      "flexDirection": "column",
      "justifyContent": "space-between",
      "padding": "0.5em",
    }
  }
/>
`;
